### PR TITLE
Fix import specifier rewriting when importing a dependency from a top-level module

### PR DESCRIPTION
**Pull request by justinfagnani**
- [x] CHANGELOG.md has been updated

### CODE REVIEW

File: src/compile-middleware.ts [hunk 1]
```diff
@@ -116,11 +116,21 @@ export function babelCompile(
 
       let transformed;
       const contentType = getContentType(response);
-      const isComponentRequest = request.baseUrl === `/${componentUrl}` &&
-          !request.path.startsWith(`/${packageName}`);
+      let requestPath = request.path;
+      const isRootPathRequest = request.path.startsWith(`/${packageName}`);
```
Comment (aomarks) [inline src/compile-middleware.ts]: i guess we should make this startWith safer too?

Comment (justinfagnani) [reply src/compile-middleware.ts]: existing code? This logic shouldn't change on windows, so I'm going to defer for now.

Comment (aomarks) [reply src/compile-middleware.ts]: just add a todo?

Comment (justinfagnani) [reply src/compile-middleware.ts]: done

**Pull request merged**